packages/react/src/components/Footer/__stories__/Footer.stories.js

```diff
@@ -1,5 +1,9 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import Footer from '../Footer';
+import { Footer } from '@ibmdotcom/react/es/components/Footer';
```

Comment:
@photodow for the stories, it should be ok to import from the relative path, since it isn't grabbing from an external es module. I think this is breaking the current netlify build actually.

Reply:
Ah, just trying to be consistent. I'll try to get that in real quick.